### PR TITLE
Changed API calls for sanity check page

### PR DESCRIPTION
**Pull request by ac-dfedko**

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/stores/ProvisioningStore.js [hunk 1]
```diff
@@ -79,6 +82,18 @@ export default class ProvisioningStore {
             }.bind(this));
     }
 
+    namespaceSetup() {
+        resetAsync(this.namespaceSetupFetchAsync, true);
+        return axios.get(API_NAMESPACE_SETUP_STEPS)
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/stores/ProvisioningStore.js]: Should be `POST /api/v1/setup`

File: ota-plus-web/app/reactapp/src/containers/Preparation.jsx [hunk 2]
```diff
@@ -52,25 +52,17 @@ class Preparation extends Component {
...
-        this.fetchTufRepoExists = this.fetchTufRepoExists.bind(this);
         this.userProfileHandler = new AsyncStatusCallbackHandler(props.userStore, 'userFetchAsync', this.checkUserProfile.bind(this));
         this.activatedProvisioningHandler = new AsyncStatusCallbackHandler(props.provisioningStore, 'provisioningStatusFetchAsync', this.checkActivatedProvisioning.bind(this));
-        this.createdTufHandler = new AsyncConflictCallbackHandler(props.packagesStore, 'tufRepoExistsFetchAsync', this.checkCreatedTuf.bind(this));
-        this.createTufHandler = new AsyncStatusCallbackHandler(props.packagesStore, 'tufRepoCreateFetchAsync', this.doubleCheckCreatedTuf.bind(this));
-        this.createdDirectorHandler = new AsyncConflictCallbackHandler(props.packagesStore, 'directorRepoExistsFetchAsync', this.checkCreatedDirector.bind(this));
-        this.createDirectorHandler = new AsyncStatusCallbackHandler(props.packagesStore, 'directorRepoCreateFetchAsync', this.doubleCheckCreatedDirector.bind(this));
+        this.createdTufHandler = new AsyncConflictCallbackHandler(props.provisioningStore, 'namespaceSetupFetchAsync', this.checkCreatedTuf.bind(this));
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/containers/Preparation.jsx]: The `setup` endpoint returns 200 with info in body, so don't need `AsyncConflictCallbackHandler`.

**Comment by ac-dfedko** [comment]
@jerrytrieu Done

File: ota-plus-web/app/reactapp/src/containers/Preparation.jsx [hunk 3]
```diff
@@ -148,10 +113,12 @@ class Preparation extends Component {
     }
 
     checkCreatedDirector() {
-        switch (this.props.packagesStore.directorRepoExistsFetchAsync.code) {
+        switch (this.props.provisioningStore.namespaceSetupFetchAsync.code) {
             case 200:
-                this.createdDirector = true;
-                this.checkedCreatedDirectorCalled = true;
+                if (this.props.provisioningStore.namespaceSetupFetchAsync.data.director) {
```
Comment (jerrytrieu) [inline ota-plus-web/app/reactapp/src/containers/Preparation.jsx]: If this is false, then just retry. And remove the non 200 status checks.

**Pull request merged**